sdcm/mgmt/common.py

```diff
@@ -18,13 +18,16 @@ def get_distro_name(distro_object):
         Distro.DEBIAN9: "debian9",
         Distro.DEBIAN10: "debian10",
         Distro.DEBIAN11: "debian11",
-        Distro.UBUNTU16: "ubuntu16",
-        Distro.UBUNTU18: "ubuntu18",
-        Distro.UBUNTU20: "ubuntu20",
-        Distro.UBUNTU22: "ubuntu22",
+        Distro.UBUNTU16_04: "ubuntu16",
+        Distro.UBUNTU18_04: "ubuntu18",
+        Distro.UBUNTU20_04: "ubuntu20",
+        Distro.UBUNTU22_04: "ubuntu22",
```

Comment:
why is it needed?

Reply:
it broke many unittests:
```
AttributeError: UBUNTU20
```

Reply:
I changed it back (changed the code to be backwrd compatible).